Podfile

```diff
@@ -120,6 +120,29 @@ target 'WooCommerce' do
   end
 end
 
+# WooCommerce POS Layer:
+# ===============
+#
+def woocommercepos_pods
+  cocoa_lumberjack
```

Comment:
I meant to ask earlier, is CocoaLumberjack included just for testing Cocoapod dependencies or for loggin like DDLog in the future?

Reply:
It was a good simple import to check that the pods project setup was working properly, but also I figured that we'd definitely want DDLog, so we won't have to undo it.